js/jsonld.js

```diff
@@ -1646,6 +1646,30 @@ jsonld.cache = {
 };
 
 /**
+ * Accept header.
+ */
+jsonld.acceptHeader = 'application/ld+json, application/json';
```

Comment:
If this isn't a default that could be changed or if we're not into figuring this out right now, let's not expose it in the public API.